sound/pci/hda/hda_intel.c

```diff
@@ -2399,6 +2399,9 @@ static const struct pci_device_id azx_ids[] = {
 	/* Icelake */
 	{ PCI_DEVICE(0x8086, 0x34c8),
 	  .driver_data = AZX_DRIVER_SKL | AZX_DCAPS_INTEL_SKYLAKE},
+	/* Tiger Lake */
+	{ PCI_DEVICE(0x8086, 0xa0c8),
+	  .driver_data = AZX_DRIVER_SKL | AZX_DCAPS_INTEL_SKYLAKE},
```

Comment:
can you also add Jasper Lake here, thanks!

Reply:
@plbossart Thanks for reminder. Forget to add this.